src/main/java/net/fabricmc/loader/impl/discovery/RuntimeModRemapper.java

```diff
@@ -78,6 +78,11 @@ public static void remap(Collection<ModCandidateImpl> modCandidates, Path tmpDir
 
 		if (modsToRemap.isEmpty()) return;
 
+		MappingConfiguration config = FabricLauncherBase.getLauncher().getMappingConfiguration();
+		String modNs = MappingConfiguration.INTERMEDIARY_NAMESPACE;
```

Comment:
maybe consider also not hardcoding this. not that I really expect anyone to want to change it, but...